fendermint/app/settings/src/lib.rs

```diff
@@ -93,6 +97,52 @@ pub struct BroadcastSettings {
     pub max_retries: u8,
 }
 
+#[derive(Debug, Deserialize, Clone)]
+pub struct TopDownConfig {
+    /// The number of blocks to delay before reporting a height as final on the parent chain.
+    /// To propose a certain number of epochs delayed from the latest height, we see to be
+    /// conservative and avoid other from rejecting the proposal because they don't see the
+    /// height as final yet.
+    pub chain_head_delay: BlockHeight,
+    /// Parent syncing cron period, in seconds
+    pub polling_interval_secs: u64,
```

Comment:
Let's use the established way here: https://github.com/consensus-shipyard/fendermint/blob/main/fendermint/app/settings/src/eth.rs#L16-L17